### DISCUSSION TITLE
CreateTranscriptionTextAsync on real time (stream)

### DISCUSSION THREAD

**Discussion by DakkuaDev** [RageAgainstThePixel/com.openai.unity#234] [Q&A] [answered]
Hello! I am using CreateTranscriptionTextAsync to make STT - TTS with Whisper API and get an audio response, but I noticed that this is the slowest part of the process. It takes around 2-3 seconds to process. Is there a way (like stream chat and stream audio) to make this iteration process faster with the API?

Thanks!

**Comment by StephenHodgson** [comment, marked as answer]
> Is there a way (like stream chat and stream audio) to make this iteration process faster with the API?

I don't believe OpenAI gives us a streaming version of this endpoint. You can chop up your audio and send parts when there is silence and process each in batches this way to get faster responses.
